### PR TITLE
create + update user - fix username bug, make organization an autocom…

### PR DESCRIPTION
**Pull request by tschia**
This change makes the organization an autocomplete based on an embedded file.
Eventually the organization list will be queried from an external service.

### CODE REVIEW

File: src/keycloak/index.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ let keycloak = Keycloak();
 let initOptions = {
     responseMode: 'fragment',
     flow: 'standard',
-    onLoad: 'login-required'
+    onLoad: 'login-required',
+    pkceMethod: 'S256'
```
Comment (sharpedavid) [inline src/keycloak/index.js:9]: I don't understand why GitHub is reporting this change. It's already on the master branch. I guess it must be reporting the diff based on where you branched? Not a big deal, just surprising and I don't understand it.

File: src/components/UserDetails.vue [hunk 2]
```diff
@@ -83,10 +92,12 @@
 
 <script>
 import UsersRepository from "@/api/UsersRepository";
+import organizations from "@/assets/organizations"
 
 export default {
   name: "UserDetails",
   props: ['userId'],
+  organizations: organizations,
```
[the remark this answers is not in the document]
Comment (tschia) [reply src/components/UserDetails.vue:100]: yep I like that better, I'll give that a try

File: src/components/UserDetails.vue [hunk 3]
```diff
@@ -49,8 +50,16 @@
...
+            <label for="org-details">Organization</label>
+            <v-autocomplete
+                id="org-details"
+                v-model="org_details"
+                :items="$options.organizations"
+                :item-text="getOrganizationText"
+                :item-value="getOrganizationValue"
+                dense
+                outlined
+            ></v-autocomplete>
```
[the remark this answers is not in the document]
Comment (tschia) [reply src/components/UserDetails.vue:62]: Hmm lets bring this back to the team - I think we want the org id and name to be tightly coupled. I'm not sure what a free text box provides in this case.

[the remark this answers is not in the document]
Comment (sharpedavid) [reply src/components/UserDetails.vue:62]: Yeah I really don't know, this is just what I thought they wanted. One possible tip off is "Business Address", do they still want that. Let's message Leon this afternoon?

**Pull request merged**